TwitchClient/Channel.swift

```diff
@@ -1,27 +0,0 @@
-//
-//  Channel.swift
-//  TwitchClient
-//
-//  Created by Ahmed kotb on 4/8/17.
-//  Copyright © 2017 Ahmed Kotb. All rights reserved.
-//
-
-import UIKit
-
-class Channel: NSObject {
-    
-    var status : String?
```

Comment:
Don't you already know all the statuses that can be sent? Wouldn't it be better if this is an enum that gets initialized from a string?